builtin/logical/pki/path_config_ca.go

```diff
@@ -48,24 +470,57 @@ func (b *backend) pathCAWrite(
 		parsedBundle.CertificateBytes = parsedBundle.IssuingCABytes
 	}
 
-	// TODO?: CRLs can only be generated with RSA keys right now, in the
-	// Go standard library. The plubming is here to support non-RSA keys
-	// if the library gets support
+	cb := &certutil.CertBundle{}
+	entry, err := req.Storage.Get("config/ca_bundle")
+	if err != nil {
+		return nil, err
+	}
+	if entry != nil {
```

Comment:
instead of all this special logic, can we have a different endpoint to support this use case? This is a pretty hairy looking